projects/igniteui-angular/src/lib/combo/combo.component.html

```diff
@@ -15,32 +15,47 @@
 </ng-template>
 
 <igx-input-group [displayDensity]="displayDensity" [type]="type" (click)="onInputClick($event)">
+    <ng-container ngProjectAs="[igxLabel]">
+        <ng-content select="[igxLabel]"></ng-content>
+    </ng-container>
+    <ng-container ngProjectAs="igx-prefix">
+        <ng-content select="igx-prefix"></ng-content>
+    </ng-container>
     <input igxInput #comboInput name="comboInput" type="text" [value]="value" readonly [placeholder]="placeholder"
-        [disabled]="disabled"/>
-    <igx-suffix *ngIf="value.length" class="clearButton" aria-label="Clear Selection" igxRipple (click)="handleClearItems($event)">
-        <igx-icon fontSet="material">clear</igx-icon>
+        [disabled]="disabled" />
+    <ng-container ngProjectAs="igx-suffix">
+        <ng-content select="igx-suffix"></ng-content>
+    </ng-container>
+    <igx-suffix *ngIf="value.length" aria-label="Clear Selection" class="igx-combo__clear-button" igxRipple (click)="handleClearItems($event)">
+        <ng-container *ngIf="clearIconTemplate">
+            <ng-container *ngTemplateOutlet="clearIconTemplate; context: {$implicit: this}"></ng-container>
```

Comment:
Similar logic applies here and this probably doesn't need any additional context at all.
Same with others like the header, add item, etc.